inst/private/python_ipc_pytave.m

```diff
@@ -61,25 +61,28 @@
 
   % TODO: bit wasteful to repeat this every call
   headers = python_header();
-  pyexec(headers)
+  pyexec(headers);
 
   % load all the inputs into python
   loc = python_copy_vars_to('_ins', true, varargin{:});
   s = strjoin(loc, newl);
-  pyexec(s)
+  pyexec(s);
 
   s = strjoin(cmd, newl);
-  pyexec(s)
+  pyexec(s);
+  
+  info.raw = pyeval('_outs[0]');
```

Comment:
hmmm, not sure.  Possibly error handling in `python_cmd`.  Otherwise, its just something that either @latot or I had when debugging `python_cmd`.